rust-peer/Cargo.toml

```diff
@@ -23,12 +23,16 @@ libp2p = { git = "https://github.com/p-shahi/rust-libp2p.git", branch = "master-
 	"rsa",
 	"ed25519",
 	"quic",
+	"request-response",
 ] }
 libp2p-webrtc = { git = "https://github.com/p-shahi/rust-libp2p.git", branch = "master-plus-webrtc-code", version = "0.4.0-alpha.4", features = [
 	"tokio",
 ] }
-libp2p-quic = {  git = "https://github.com/p-shahi/rust-libp2p.git", branch = "master-plus-webrtc-code", features = ["tokio"] }
+libp2p-quic = { git = "https://github.com/p-shahi/rust-libp2p.git", branch = "master-plus-webrtc-code", features = [
```

Comment:
I'd be great to avoid this reformatting :)

Reply:
91807fb